crates/sncast/src/starknet_commands/helpers/fee.rs

```diff
@@ -10,16 +10,16 @@ pub struct FeeArgs {
     #[clap(long)]
     pub fee_token: Option<FeeToken>,
 
-    /// Max fee for the transaction. If not provided, will be automatically estimated. (Only for ETH fee payment)
+    /// Max fee for the transaction. If not provided, will be automatically estimated
     #[clap(short, long)]
     pub max_fee: Option<FieldElement>,
 
     /// Max gas amount. If not provided, will be automatically estimated. (Only for STRK fee payment)
-    #[clap(long)]
+    #[clap(long, conflicts_with = "max_fee", requires = "max_gas_unit_price")]
```

Comment:
I think it's not that simple (the conflicting/requiring part)
If we're able to estimate both of those (gas/gas price), we should be able to pass just one of those and estimate the other